softwareComponents/configuration/include/configuration/rofiworld.hpp

```diff
@@ -445,9 +445,14 @@ class SimpleCollision {
     bool operator()( const Module& a, const Module& b, Matrix posA, Matrix posB ) {
         for ( auto pA : a.getOccupiedRelativePositions() ) {
             for ( auto pB : b.getOccupiedRelativePositions() ) {
-                if ( rofi::configuration::matrices::equals( static_cast< Matrix >( posA * pA ), posB * pB ) ) {
+
+                Vector A = static_cast< Matrix >( posA * pA ) * Vector{ 0, 0, 0, 1 };
```

Comment:
Použij `rofi::configuration::matrices::center` místo násobení vektorem

Reply:
Máš pravdu, potřebné funkce tam opravdu jsou; asi jsou to i dobří kandidáti na přesunutí do knihovny geometry, až bude existovat (respektive mi dává větší smysl mít je tam než v konfiguraci).